unitary/alpha/quantum_world_test.py

```diff
@@ -151,3 +151,59 @@ def test_pop_not_enough_reps():
     results = board.peek([lights[16]], count=200)
     assert len(results) == 200
     assert all(result[0] == Light.GREEN for result in results)
+
+
+def test_get_histogram_and_get_probabilities():
+    l1 = alpha.QuantumObject("l1", Light.GREEN)
+    world = alpha.QuantumWorld([l1])
+    histogram = world.get_histogram()
+    assert histogram == [{0:0,1:100}]
+    probs = world.get_probabilities()
+    assert probs == [{0:0.0,1:1.0}]
+    bin_probs = world.get_binary_probabilities()
+    assert bin_probs == [1.0]
+    alpha.Flip()(l1)
+    histogram = world.get_histogram()
+    assert histogram == [{0:100,1:0}]
+    probs = world.get_probabilities()
+    assert probs == [{0:1.0,1:0.0}]
+    bin_probs = world.get_binary_probabilities()
+    assert bin_probs == [0.0]
+    alpha.Superposition()(l1)
+    histogram = world.get_histogram()
+    assert len(histogram)==1
+    assert len(histogram[0])==2
+    assert histogram[0][0]>10
+    assert histogram[0][1]>10
+    probs = world.get_probabilities()
+    assert len(probs)==1
+    assert len(probs[0])==2
+    assert probs[0][0]>.1
+    assert probs[0][1]>.1
+    bin_probs = world.get_binary_probabilities()
+    assert 0.1 <= bin_probs[0] <= 1.0
+
+    l1 = alpha.QuantumObject("l1", StopLight.YELLOW)
```

Comment:
Can we break this up into multiple tests?  It might make it more obvious what you are testing, and they seem independent.

Reply:
Done.